documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -577,35 +577,14 @@ WildFly will by default attempt to guess which Persistence Provider you need by
 ==== Enable support for JEE 8
 
 Hibernate OGM {hibernate-ogm-version} requires **CDI 2.0** and **JPA 2.2**, that belong to **JEE8** specification.
-WildFly 12 has limited support for Java EE8.
+WildFly 13 has support for Java EE8. But in order to enable a JEE8 profile, we need to set a system property.
```

Comment:
This is not an improvement. The support is still limited AFAIK, at least, WF 13 is not certified so I don't think we can say if supports EE 8.

By the way I think it's written "EE 8".

The second sentence is redundant with the sentence below so I don't think you should add it.

Reply:
Reading the release page: http://wildfly.org/news/2018/05/30/WildFly13-Final-Released/ I'm thinking that maybe we can remove the **limited** world. But as you prefer, is not critical in my opinion. Ok s/EE8/EE 8. Moreover I agree the latter sentence is redundant.

Reply:
Yeah OK about `limited`.